src/lib/routes/edge-api/index.ts

```diff
@@ -59,12 +63,15 @@ export default class EdgeController extends Controller {
             middleware: [
                 this.openApiService.validPath({
                     tags: ['Edge'],
+                    summary:
+                        'Filter a list of tokens returning only the valid ones',
+                    description:
+                        'Accepts a list of tokens, and returns those that are valid with the projects they can access',
```

Comment:
What does "valid with the projects they can access" mean? Is it "valid with the projects" or does it return the tokens with their projects?

But ooh, this one is tricky. What about something like this? Trying to be as clear and explicit as possible. But it might not be necessary to describe in the description **everything** that is included in the edge-token-schema 🤔 

```suggestion
                    summary:
                        'Check which tokens are valid',
                    description:
                        'This operation accepts a list of tokens to validate. Unleash will validate each token you provide. For each valid token you provide, Unleash will return the token along with its type and which projects it has access to.',
```